README.md

```diff
@@ -41,14 +41,32 @@ $Env:Path += ";$(pwd)"
 
 ## Getting the Code and Building Electron
 
-After installing build-tools, you can run a new Electron build with this command:
+After installing build-tools, you'll need to **create a new directory** for the Electron build.
+In your terminal, you'll need to run something like:
+
+```sh
+# create directory
+mkdir electron-gn
+
+# get its path (on Unix):
+cd electron-gn && pwd
+
+# get its path (on Windows):
+cd electron-gn && cd
+```
+
+in the directory where you typically place your code projects.
+
+After you've created this new directory, you can run a new Electron build with this command:
 
 ```sh
 # The 'Hello, World!' of build-tools: get and build `master`
 
 # On macOS if you aren't sure you have the right SDK version you should run
 e load-macos-sdks
 
+# REPLACE /path/to/new/electron/directory with the path to the directory
+# you created in the command you ran above
```

Comment:
:+1: on adding this clarification, but needs to be reworked a little if the previous section is removed (see comment above).

Maybe something like

```
# Choose where where to keep Electron's source and build files.
# You can choose any naming scheme you like. Two options are
# * `--init=~/electron-gn` if you're only going to use one branch
# * `--init=~/electron-gn/branch` (e.g. `~/electron-gn/master`) if you're going to use multiple branches
```

?